kafka/source/pkg/adapter/adapter_test.go

```diff
@@ -628,3 +629,48 @@ func TestAdapterStartFailure(t *testing.T) {
 
 	_ = adapter.Start(make(chan struct{}))
 }
+
+func TestSplitOnCommas(t *testing.T) {
+	for _, tt := range []struct {
+		name             string
+		bootstrapServers string
+		want             []string
+	}{
+		{
+			"two commas, one url",
+			",my-cluster-kafka-bootstrap.kafka:9092,",
+			[]string{
+				"my-cluster-kafka-bootstrap.kafka:9092",
+			},
+		},
+		{
+			"two commas, two urls",
+			",url-one.kafka:9092,url-two.kafka:9092",
+			[]string{
+				"url-one.kafka:9092",
+				"url-two.kafka:9092",
+			},
+		},
+		{
+			"two commas, no url",
+			",,",
+			[]string{},
+		},
+		{
+			"two commas, three urls",
+			"url-one.kafka:9092,url-two.kafka:9092,url-three.kafka:9092",
+			[]string{
+				"url-one.kafka:9092",
+				"url-two.kafka:9092",
+				"url-three.kafka:9092",
+			},
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			got := SplitOnCommas(tt.bootstrapServers)
+			if diff := cmp.Diff(tt.want, got); diff != "" {
```

Comment:
just wondering why you used `cmp` instead of `reflect.DeepEqual` ?

Reply:
Path of least surprise in the codebase
```
% git grep -i cmp.diff | grep -v ^vendor | grep _test.go | wc -l
63
% git grep -i reflect.deep | grep -v ^vendor | grep _test.go | wc -l
0

Reply:
ah didn't realize we were using it in other spots - thanks for the info